ui/src/components/core/color-picker/color-picker.tsx

```diff
@@ -38,9 +40,9 @@ export const ColorPicker: React.FC<ColorPickerProps> = ({
 
   return (
     <Card.Text css={{ height: '$22', mt: '$6' }}>
-      <div className="relative">
-        <Flex css={{ gap: '$3h', alignItems: 'center' }}>
-          <span>Primary Color</span>
+      <CS.Container>
+        <Flex css={{ gap: '1px' }}>

```

Comment:
lets avoid using `px` in these cases do like `(value in pixels)/16` for `rem`. 16 is our base value and when go to mobile it will fill better.
```suggestion
        <Flex css={{ gap: '0.0625rem' }}>
```